apps/web/src/components/GalleryEditor/PiecesSidebar/SidebarTokens.tsx

```diff
@@ -121,7 +148,7 @@ export const SidebarTokens = ({
 
   const rows = useMemo(() => {
     if (shouldUseCollectionGrouping) {
-      const groups = groupCollectionsByAddress({ tokens });
+      const groups = groupCollectionsByAddress({ filteredTokensBySelectedWallet });
```

Comment:
looks like there's another usage of `groupCollectionsByAddress` on line 187 where we'll need to reflect the prop name change